### PR TITLE
auth docs: fix one error and expand a bit on queues and threads

### PR DESCRIPTION
**Pull request by Habbie**
### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: docs/performance.rst [hunk 1]
```diff
@@ -62,6 +62,11 @@ To determine if PowerDNS is unable to keep up with packets, determine
 the value of the :ref:`stat-qsize-q` variable. This represents the number of
 packets waiting for database attention. During normal operations the
 queue should be small.
+This number is a total over all receiver threads.
+
+The :ref:`setting-max-queue-length` and :ref:`setting-overload-queue-length` settings determine how PowerDNS deals with growing queues.
+If the queue for a single receiver thread (and its associated distributor threads) grows beyond the ``overload`` number, queries are answered only from the packet cache so the database can hopefully recover.
```
Comment (omoerbeek) [inline docs/performance.rst:68]: When reading this, an immediate question popped up in my mind: what happens to packet cache misses in the overload case? Arte they dropped or maybe answered with ServFail???

File: docs/performance.rst [hunk 2]
```diff
@@ -62,6 +62,11 @@ To determine if PowerDNS is unable to keep up with packets, determine
...
+If we reach the ``max`` number, we consider the situation hopeless and respawn.
```
Comment (omoerbeek) [inline docs/performance.rst:69]: Make it clear the whole proces is respawned, not just a single thread.
```suggestion
If we reach the ``max`` number, we consider the situation hopeless and respawn the server process.
```

File: docs/settings.rst [hunk 3]
```diff
@@ -1228,6 +1228,7 @@ will generally suffice for most installations.
 
 If this many packets are waiting for database attention, consider the
 situation hopeless and respawn.
```
Comment (omoerbeek) [inline docs/settings.rst:1230]:
```suggestion
situation hopeless and respawn the server process.
```

File: docs/performance.rst (hunk 1 above)
Comment (Habbie) [reply docs/performance.rst:68]: the settings page (linked) answers that: they are dropped

Comment (omoerbeek) [reply docs/performance.rst:68]: OK, that's fine.

**Review by rgacogne — APPROVED** [review]

**Review by omoerbeek — APPROVED** [review]

**Pull request merged**